packages/config/src/load.js

```diff
@@ -18,7 +18,7 @@ export async function loadNuxtConfig ({
   configFile = defaultNuxtConfigFile,
   configContext = {},
   configOverrides = {},
-  createRequire = module => isJest ? _createRequire(module.filename) : esm(module)
+  createRequire = module => isJest ? _createRequire(module.filename) : esm(module, { cache: false, cjs: { cache: false } })
```

Comment:
`cjs.cache` option is a flag to indicate where to expose runtime cache not about fs cache: https://github.com/standard-things/esm/blob/950e74186acf7e2e733168b327f8d9747ad13427/src/module/esm/load.js#L58

Reply:
I didn't test but as reported by @rchl (https://github.com/nuxt/nuxt.js/issues/7838#issuecomment-667934798) we may set `cache` option to `node_modules/.cache/esm` to avoid regression and still benefit fs based caching.

Reply:
I'm not sure about that. The issue seems to be about npm not clearing the cache. I'm not sure I understand how having it in one place help the issue.

Reply:
I think cache path of esm is calculated by esm package dir which can not be override

Reply:
> `cjs.cache` option is a flag to indicate where to expose runtime cache not about fs cache: [standard-things/esm@`950e741`/src/module/esm/load.js#L58](https://github.com/standard-things/esm/blob/950e74186acf7e2e733168b327f8d9747ad13427/src/module/esm/load.js#L58)

Which will be then written to disk on exit no?

https://github.com/standard-things/esm/blob/91a154bedd2700a8fd85d78c1e15161676104419/src/caching-compiler.js#L168

(I don't pretend to understand esm's code. This is just after quite look.)

Reply:
@clarkdo You are right it is calculated by esm [__dirname](https://github.com/standard-things/esm/blob/91a154bedd2700a8fd85d78c1e15161676104419/esm.js#L105)

@rchl Yes. It was about `cjs.cache` option which indicates using `require.cache{}` or another internal object for in mem cache (when require same file multiple times)